tests/lib/rules/camelcase.js

```diff
@@ -79,6 +79,30 @@ ruleTester.run("camelcase", rule, {
         {
             code: "import { no_camelcased as camelCased, anoterCamelCased } from \"external-module\";",
             parserOptions: { ecmaVersion: 6, sourceType: "module" }
+        },
+        {
+            code: "const { no_camelcased } = bar;",
+            options: [{ properties: "never" }],
```

Comment:
I think we can remove the options for these tests and then maybe have one test for `properties: "never"` and one for `properties: "always"`. While they're not wrong, I think the current options in these tests signal that it should be configured with `"never"` (which is no longer true).